stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -229,6 +232,12 @@ public fun Messages(
             onLastVisibleMessageChanged(message.message)
         }
     }
+
+    /** Clean up: Pause any playing audio tracks in onPause(). **/
+    LifecycleEventEffect(Lifecycle.Event.ON_PAUSE) {
+        val audioPlayer = ChatClient.instance().audioPlayer
+        audioPlayer.pause()
```

Comment:
Our UI should't use the ChatClient itself.
Could we move this logic to the ViewModel?